test/test_datasources.py

```diff
@@ -128,24 +111,24 @@ def test_import_layer(self):
 class TestNetcdfGroundwaterDataSource(unittest.TestCase):
     def test_constructor(self):
         """Test empty constructor."""
-        NetcdfGroundwaterDataSource()
+        ThreediResult()
 
     def test_sanity(self):
-        nds = NetcdfGroundwaterDataSource()
+        nds = ThreediResult()
```

Comment:
nds is de afko v/d oude class.

Ik zou er gewoon (in de hele file) `threedi_result` van maken.